_inc/client/state/jumpstart/reducer.js

```diff
@@ -3,6 +3,7 @@
  */
 import { combineReducers } from 'redux';
 import assign from 'lodash/assign';
+import { createHistory } from 'history';
```

Comment:
@eliorivero this is breaking the tests because they don't run with real DOM and apparently createHistory is demanding it.

Also, we shouldn't be updating anything else but the state from inside a reducer. 

This could be implemented inside one of the top-level components. I'll try to dig the issue or an alternative a little further